nexus/src/external_api/console_api.rs

```diff
@@ -316,7 +316,7 @@ pub async fn login(
 /// data (like a SAMLResponse). Use these to set the user's session cookie.
 #[endpoint {
    method = POST,
-   path = "/login/{silo_name}/{provider_name}",
+   path = "/login/{silo_name}/saml/{provider_name}",
```

Comment:
This is the URL that will have to change in the Keycloak configuration, it's the path the IdP POSTs to.

Reply:
Let's also maybe update the remote access docs to use a different name for the provider instead of `saml`, lest we end up with the repetitive `/login/mercury/saml/saml` 😅